cluster/ephemeral-provider-common.sh

```diff
@@ -29,11 +29,18 @@ function _registry_volume() {
 }
 
 function _add_common_params() {
-    local params="--nodes ${KUBEVIRT_NUM_NODES} --memory 4096M --cpu 4 --random-ports --background --prefix $provider_prefix --registry-volume $(_registry_volume) kubevirtci/${image} ${KUBEVIRT_PROVIDER_EXTRA_ARGS}"
+    local params="--nodes ${KUBEVIRT_NUM_NODES} --cpu 4 --random-ports --background --prefix $provider_prefix --registry-volume $(_registry_volume) kubevirtci/${image} ${KUBEVIRT_PROVIDER_EXTRA_ARGS}"
```

Comment:
why not keeping it similar to original and just use --memory ${KUBEVIRT_MEMORY_SIZE}

Reply:
I would suggest stay with the 4096M as sane default. We have allocated 12GB by default in stdci and We are running 2 nodes, with 8GB it would go over the limit.

Also it would be better as @michalskrivanek noted, direct use of variable similar as {KUBEVIRT_NUM_NODES} is cleaner approach and avoids later use of ifs.